string commandType
geometry_msgs/Point32 nextPoint
float32 speed
float32 acceleration
int32 gripper
int32 zAxisCommand
int32 inputNumber
int32 waitTime
